src/github.com/getlantern/flashlight/client/balancer.go

```diff
@@ -55,26 +44,15 @@ func (client *Client) initBalancer(cfg *ClientConfig) {
 	}
 
 	bal := balancer.New(dialers...)
-
-	if client.balInitialized {
-		log.Trace("Draining balancer channel")
-		old := <-client.balCh
+	oldBal, ok := client.bal.Get(25 * time.Millisecond)
```

Comment:
Any reason not to just make the wait time here 0 @oxtoacart? Seems like waiting at all just introduces an unnecessary delay.